src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -404,6 +404,158 @@ def test_disp_s1_pge_validate_inputs(self):
         for f in files_to_remove:
             os.remove(f)
 
+    def test_get_cslc_input_burst_id_set(self):
+        """
+        Set of tests to sanity check the burst ideas associated with:
```

Comment:
should this be "burst ids" instead of "burst ideas"?

Reply:
haha!  oh brother, I can't believe I let that one by.